api/spec/controllers/spree/api/v2/base_controller_spec.rb

```diff
@@ -46,4 +46,36 @@ def default_resource_includes
       end
     end
   end
+
+  describe '#sparse_fields' do
+    context 'not passed in params' do
+      before do
+        dummy_controller.params = {}
+      end
+
+      it 'returns nil' do
+        expect(dummy_controller.send(:sparse_fields)).to eq(nil)
+      end
+    end
+
+    context 'with invalid params format' do
+      before do
+        dummy_controller.params = { fields: 'variants,images,taxons' }
```

Comment:
@caudatecoder shouldn't this be stuff like `slug`, `name` etc?

Reply:
Yeah it will be closer to the real payload